.github/workflows/linux-other-builds.yml

```diff
@@ -103,3 +103,48 @@ jobs:
           CTEST_OUTPUT_ON_FAILURE: 1
       working-directory: ${{runner.workspace}}/build
       run: ctest -C ${{matrix.build_type}} -j `nproc` ${{matrix.other_ctest_args}}
+  clang-tidy:
+    name: clang-tidy-${{matrix.version}}, ${{matrix.build_description}}, C++${{matrix.std}} ${{matrix.build_type}}
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          # Extras and examples with clang-tidy-10
+          - version: "10"
+            tidy_options: ''
+            build_description: Extras + Examples
```

Comment:
```suggestion
            build_description: clang-tidy
```
instead?

Reply:
"clang-tidy" is already included in the job description. I've replaced this by "all", but it's not really _completely_ all. I guess it's hard to put variant-specific description when there is just one variant.

Reply:
Right, then "extras + examples" is fine.